docs/tutorials/tutorial-rollup.md

```diff
@@ -24,18 +24,21 @@ sidebar_label: Aggregate data with rollup
   -->
 
 
-Apache Druid can summarize raw data at ingestion time using a process we refer to as "rollup". Rollup is a first-level aggregation operation over a selected set of columns that reduces the size of stored data.
+Apache Druid&circledR; can summarize raw data at ingestion time using a process known as "rollup." [Rollup](../ingestion/rollup.md) is a first-level aggregation operation over a selected set of columns that reduces the size of stored data.
```

Comment:
Link to the MSQ rollup doc (https://druid.apache.org/docs/latest/multi-stage-query/concepts/#rollup) instead but use relative path.

Reply:
relative link added